pkgs/development/tools/build-managers/bazel/bazel_3/default.nix

```diff
@@ -527,6 +528,14 @@ stdenv.mkDerivation rec {
         --output=./bazel_src/output/bazel-complete.bash \
         --prepend=./bazel_src/scripts/bazel-complete-header.bash \
         --prepend=./bazel_src/scripts/bazel-complete-template.bash
+
+    # need to change directory for bazel to find the workspace
+    cd ./bazel_src
+    # build execlog tooling
+    export HOME=$(mktemp -d)
+    ./output/bazel build  src/tools/execlog:all
+    ./output/bazel build  src/tools/execlog:parser_deploy.jar
```

Comment:
Technically, you only need to build the `src/tools/execlog:parser_deploy.jar` target. But should not hurt if you keep the first command.

Reply:
indeed, I thought about removing it but it's such a small build. Your comment lead me to remove it.